conda_build/build.py

```diff
@@ -1821,9 +1821,10 @@ def _write_sh_activation_text(file_handle, m):
                             cygpath_suffix))
 
     if conda_46:
-        file_handle.write("eval \"$('{sys_python}' -m conda shell.bash hook)\"\n".format(
-            sys_python=sys.executable,
-        ))
+        py_flags = '-m' if m.config.debug else '-m -I'
```

Comment:
Looks like it needs to be `-I -m` https://docs.python.org/3/using/cmdline.html#command-line

Reply:
Good catch!
```suggestion
        py_flags = '-m' if m.config.debug else '-I -m'
```